source/common/event/dispatched_thread.cc

```diff
@@ -0,0 +1,50 @@
+#include "common/event/dispatched_thread.h"
+
+#include <chrono>
+
+#include "envoy/event/dispatcher.h"
+#include "envoy/event/timer.h"
+#include "envoy/server/configuration.h"
+
+#include "common/api/api_impl.h"
+#include "common/common/thread.h"
+
+#include "spdlog/spdlog.h"
+
+namespace Envoy {
+namespace Event {
+
+void DispatchedThreadImpl::start(Server::GuardDog& guard_dog) {
```

Comment:
No (neither does Worker, which I was modeling this after).  This is a pretty thin wrapper around Dispatcher, I think for any case where you want mocking what you'd do is add a ctor that takes a mock Dispatcher and use that.  Worker doesn't have this functionality currently.